vivo-project/Vitro
# [ i18n sprint ] Decouple email templates and translations

[Companion Vitro-languages PR](https://github.com/vivo-project/Vitro-languages/pull/61)
Modified FreemarkerEmailMessage and classes that used it:
Removed EmailDirective and error-email.ftl
Send email in separate thread to prevent web interface delays.
With this PR translations, extracted from email templates into property files could be used.
#How to test#
1. Configure email settings
For each of installed languages execute following actions:
3. Create new user, e-mail to the user about created account should be received.
4. Use the link from the e-mail, set user password, you should receive e-mail after password is set.
5. Try reset user's password, e-mail to the user about created account should be received.
6. Try to log in as a user and change e-mail address, you should receive e-mail about that.
7. Break your page-home.ftl and open home page, you should see "There was an error in the system. This error has been reported to the site administrator. " Check email in administrator's mailbox

# Interested parties
 @VIVO-project/vivo-committers
